src/main/java/tornadofx/Controls.kt

```diff
@@ -254,84 +439,40 @@ fun <T> ToggleGroup.bind(property: ObservableValue<T>) = selectedValueProperty<T
- */
-fun EventTarget.togglebutton(
-        text: String? = null,
-        group: ToggleGroup? = getToggleGroup(),
-        selectFirst: Boolean = true,
-        value: Any? = null,
-        op: ToggleButton.() -> Unit = {}
-) = ToggleButton().attachTo(this, op) {
-    it.text = if (value != null && text == null) value.toString() else text ?: ""
-    it.properties["tornadofx.toggleGroupValue"] = value ?: text
-    if (group != null) it.toggleGroup = group
-    if (it.toggleGroup?.selectedToggle == null && selectFirst) it.isSelected = true
-}
-
-fun EventTarget.togglebutton(
-        text: ObservableValue<String>? = null,
-        group: ToggleGroup? = getToggleGroup(),
-        selectFirst: Boolean = true,
-        value: Any? = null,
-        op: ToggleButton.() -> Unit = {}
-) = ToggleButton().attachTo(this, op) {
-    it.textProperty().bind(text)
-    it.properties["tornadofx.toggleGroupValue"] = value ?: text
-    if (group != null) it.toggleGroup = group
-    if (it.toggleGroup?.selectedToggle == null && selectFirst) it.isSelected = true
-}
-
 fun ToggleButton.whenSelected(op: () -> Unit) {
     selectedProperty().onChange { if (it) op() }
 }
 
-/**
- * Create a radiobutton inside the current or given toggle group. The optional value parameter will be matched against
- * the extension property `selectedValueProperty()` on Toggle Group. If the #ToggleGroup.selectedValueProperty is used,
- * it's value will be updated to reflect the value for this radio button when it's selected.
- *
- * Likewise, if the `selectedValueProperty` of the ToggleGroup is updated to a value that matches the value for this
- * radiobutton, it will be automatically selected.
- */
-fun EventTarget.radiobutton(
-        text: String? = null,
-        group: ToggleGroup? = getToggleGroup(),
-        value: Any? = null,
-        op: RadioButton.() -> Unit = {}
-) = RadioButton().attachTo(this, op) {
-    it.text = if (value != null && text == null) value.toString() else text ?: ""
-    it.properties["tornadofx.toggleGroupValue"] = value ?: text
-    if (group != null) it.toggleGroup = group
-}
 
-fun EventTarget.label(text: String = "", graphic: Node? = null, op: Label.() -> Unit = {}) = Label(text).attachTo(this, op) {
+// ================================================================
+// Label
+
+fun EventTarget.label(
+    text: String? = "",
+    graphic: Node? = null,
+    op: Label.() -> Unit = {}
+): Label = Label(text).attachTo(this, op) {
     if (graphic != null) it.graphic = graphic
 }
 
 inline fun <reified T> EventTarget.label(
-        observable: ObservableValue<T>,
-        graphicProperty: ObjectProperty<Node>? = null,
-        converter: StringConverter<in T>? = null,
-        noinline op: Label.() -> Unit = {}
-) = label().apply {
+    observable: ObservableValue<T>,
+    graphicProperty: ObservableValue<Node>? = null, // FIXME Inconsistent with all the other functions. There is no other function with ObservableValue<Node>
```

Comment:
Should this be `graphic: Node` like the rest of functions, or should all other functions have `graphicProperty: ObservableValue<Node>?`?